docs/pages/includes/install-linux.mdx

```diff
@@ -1,22 +1,24 @@
-<ScopedBlock scope={["enterprise"]}>
-
-Visit your [Teleport account](https://teleport.sh) and select the URL for your package of choice.
-
-</ScopedBlock>
+Use the appropriate commands for your environment to install your package.
 
-<ScopedBlock scope={["cloud"]}>
+<Admonition type="note">
+`apt` and `yum` repos don't expose packages for all distribution variants.
+When following installation instructions, you might need to replace `ID` with
+`ID_LIKE` to install packages of the closest supported distribution.
 
-Visit the [Downloads Page](/docs/choose-an-edition/teleport-cloud/downloads.mdx)
-and select the URL for your package of choice.
+Currently supported distributions (and `ID`) are:
+- RHEL (`rhel`)
+- CentOS (`centos`)
+- Debian (`debian`)
+- Ubuntu (`ubuntu`)
+- AmazonLinux (`amzn`)
```

Comment:
Should we specify versions here as well? RHEL/CentOS >=7, Debian >= 8, Ubuntu > 14.04, Amazon Linux 2 and 2023?

Reply:
Will do. Ubuntu 14.04 is an issue because we don't support it through apt repos. It can be installed through tarball, or direct pkg unpack. This admonition is about apt/yum. I can add a note dedicated to ubuntu 14.04 install or add new tab (it won't be visible because of horizontal scrolling though)

Reply:
That's fine let's just say the oldest which we do publish in repos then (16.04?)